### ISSUE TITLE
Prevent aspect ratio from switching when image is rotated (using a locked aspect)

### ISSUE THREAD

**Issue by carman247** [hnvn/flutter_image_cropper#240]
Why does the aspect ratio (if locked) switch on iOS when the image is rotated? Is there any way to prevent this?

e.g.

I want to use a locked aspect ratio, and only one. The problem is when the user rotates the image the aspect switches (16:9 becomes 9:16) ...

It's great that the image can be rotated but if the aspect flips at the same time then that kind of defeats the point in having a locked aspect ... 

This is only a problem with iOS

**Comment by jithinvarghese** [comment]
Did you found any solution for this?

**Comment by carman247** [comment]
> Did you found any solution for this?

No, not yet unfortunately. I've had to disable rotation on iOS for now.

**Comment by duo-nv** [comment]
We had the same issue and added this fix to a fork:
https://github.com/duo-nv/flutter_image_cropper/commit/2aab8aac77dce271e2d0f8283e6ff4a0e972e0e8

We haven't tested the other features, but this worked for our use case.

**Comment by mtical** [comment]
Any concrete update on this?

**Comment by mtical** [comment]
FYI - The fix works with the latest (1.4.0 version). Doesn't seem to work with prior versions.

**Comment by mukkoticf** [comment]
How to disable rotation in iOS?
